web/themes/custom/pineapple/sass/components/paragraphs/_image-grid.scss

```diff
@@ -0,0 +1,50 @@
+.field--name-field-p-ig-item {
```

Comment:
I think you should update this to a custom class, i think we need to start making a steer away from 
drupal specific classes where we can (when we've made custom templates).

Reply:
Okay sure, I'll add the refactor of the other templates to my bugfix to do list.